src/app/frontend/common/components/label/label_directive.js

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import LabelController from './label_controller';
+
+/**
+ * Returns directive definition for label.
+ *
+ * @return {!angular.Directive}
+ */
+export default function labelDirective() {
```

Comment:
Is this **a** label or multiple labels? This should be reflected in the directive name.